java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -6693,6 +6693,9 @@ void testApplyBooleanMaskFromListOfStructure() {
     }
   }
 
+  /**
+   * The caller needs to make sure to close the returned ColumnView
```

Comment:
```suggestion
   * The caller needs to make sure to call this function in a `try` block as local variables need to be closed.
```

Reply:
ugh, I am rushing through this and context-switching isn't helping either. I think it will make sense for me to close all the local allocations and leave only the returned vector to be closed by the caller.